src/OrbitGl/StringManager.h

```diff
@@ -16,9 +16,12 @@
 namespace orbit_gl {
 
 // This class is a thread-safe wrapper around `absl::flat_hash_map<uint64_t, std::string>`.
+// StringManager has a global instance that can be used in a singleton-style (GetInstance()),
+// but instantiating a new instance of StringManager is allowed as well.
 class StringManager {
  public:
   StringManager() = default;
+  static StringManager* GetInstance();
```

Comment:
Isn't this effectively making StringManager global?

Reply:
Well, it's effectively a singleton, so we could argue if that is the accepted way of "global" ;-)
But anyways, I've discussed this with Henning and will be removing this change from the PR. It's no longer needed to make the current tests work, and I was wrong about how StringManager is used, it's not a good idea to have this as a Singleton regardless of the tests.